src/_pytest/_shtab.py

```diff
@@ -0,0 +1,13 @@
+from argparse import Action
```

Comment:
we either add a note here that this is a shim, or move all stab imports here

Reply:
OK. `"""A shim of shtab."""` in docstring should be enough.